@xen-orchestra/web/src/components/site/dashboard/Backups.vue

```diff
@@ -27,7 +27,11 @@ import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
 import { computed } from 'vue'
 import { useI18n } from 'vue-i18n'
 
-const { record, isReady } = useDashboardStore().subscribe()
+const { record } = useDashboardStore().subscribe()
+
+const backupsIsReady = computed(
```

Comment:
```suggestion
const areBackupsReady = computed(
```
seems more correct, no ?

Reply:
Yes. I will update all `*IsReady` to `are*Ready`